pkg/apiserver/registry/virtualmachinepolicy/rest.go

```diff
@@ -134,7 +134,7 @@ func (r *REST) convertToVMP(internal *cloud.NetworkPolicyStatus) *runtimev1alpha
 	inProgress := false
 	npStatusList := make(map[string]*runtimev1alpha1.NetworkPolicyStatus)
 	for anp, status := range internal.NPStatus {
-		if status == cloud.NetworkPolicyStatusApplied {
+		if strings.Contains(status, cloud.NetworkPolicyStatusApplied) {
```

Comment:
Is the get vmp output updated with this change?

Reply:
no, still shown as SUCCESS